### PR TITLE
Added 404 NotFound page

### LINKED ISSUE #171: Make 404 Page
There doesn't seem to be a 404 page on the website at the moment.
I would like to see whether I can mock-up a quick version.

### PR DESCRIPTION
**Pull request by michaelkebe**
Added a 404 Not Found Page. Fixes #171.

### CODE REVIEW

**Comment by peerchemist** [comment]
Looks good to me. Thanks.

@bananenwilly how about you?

**Comment by bananenwilly** [comment]
310 lines of new SCSS for a 404 page are not really acceptable

**Review by bananenwilly — APPROVED** [review]

**Review by bananenwilly — CHANGES_REQUESTED** [review]
reduce SCSS

File: src/pages/NotFound/NotFound.jsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import { useTranslation } from 'react-i18next';
+import './NotFound.scss';
+import Menu from '../../components/Menu/Menu';
+import Loader from '../../components/Loader/Loader';
+
+function NotFound() {
+  const { t } = useTranslation();
+
+  return (
+    <>
```
Comment (bananenwilly) [inline src/pages/NotFound/NotFound.jsx:11]: nice

**Review by bananenwilly — APPROVED** [review]

**Comment by bananenwilly** [comment]
@michaelkebe  thanks and welcome 👍🏻

**Pull request merged**